src/main/java/net/javadiscord/javabot/systems/help/commands/subcommands/ExperienceLeaderboardSubcommand.java

```diff
@@ -63,17 +64,18 @@ public static void handleButtons(ButtonInteractionEvent event, String[] id) {
 	}
 
 	private static MessageEmbed buildExperienceLeaderboard(Guild guild, HelpAccountRepository dao, int page) throws SQLException {
-		int maxPage = dao.getTotalRankedAccounts() / PAGE_SIZE;
-		List<HelpAccount> accounts = dao.getAccountsWithRank(Math.min(page, maxPage), PAGE_SIZE);
+		int maxPage = dao.getTotalAccounts() / PAGE_SIZE;
+		List<HelpAccount> accounts = dao.getAccounts(Math.min(page, maxPage), PAGE_SIZE);
 		EmbedBuilder builder = new EmbedBuilder()
 				.setTitle("Experience Leaderboard")
 				.setColor(Bot.config.get(guild).getSlashCommand().getDefaultColor())
 				.setFooter(String.format("Page %s/%s", Math.min(page, maxPage), maxPage));
 		accounts.forEach(account -> {
 			Map.Entry<Long, Double> currentRole = account.getCurrentExperienceGoal(guild);
+			System.out.println(guild.getJDA().getUserCache().toString());
```

Comment:
Don't print debug output to `System.out` in production. if that should be kept, use a logger.

Reply:
Oh, lmao. Forgot to remove that

Reply:
This was already removed in 11bf6bd7ef8a35f9a1fe4fdec7aadb593a690a2b